source/user-manual/capabilities/file-integrity/how-it-works.rst

```diff
@@ -3,26 +3,16 @@
 How it works
 ============
 
+Since v3.12, Wazuh uses the new FIM module. To learn about the functionality of the FIM used in the previous Wazuh releases, please refer to the corresponding version of the documentation. 
```

Comment:
![image](https://user-images.githubusercontent.com/3830293/81082839-1a24b880-8ef4-11ea-8262-f2794ccdfc37.png)

Both paragraphs need improvements. 
1. You should indicate a version, v3.12 is not a version, is a development branch. v3.12.0 is more accurate. 
2. Both start with the same `Since` sentence. I would start the first by introducing the new FIM module (as you did), presenting the following diagram that will explain how the new FIM works. 
3. Removing the Since Wazuh v3.12 from the second paragraph.
4. Slit the paragraph from `The new FIM synchronization mechanism ensures` could works, because it explaining a second part of the behavior and we avoid a too big paragraph, making easy the reading.

Reply:
Done